packages/blockchain-wallet-v4/src/coinSelection/coin.js

```diff
@@ -48,6 +30,20 @@ export class Coin extends Type {
   isFromLegacy () {
     return !this.isFromAccount()
   }
+  type () {
+    try {
+      switch (this.address[0]) {
+        case '1':
+          return 'P2PKH'
+        case '3':
+          return 'P2SH-P2WPKH'
+        default:
+          return 'P2PKH'
+      }
+    } catch (e) {
+      return 'P2PKH'
+    }
+  }
```

Comment:
Is the best way to do this?

Reply:
What about native segwit? Is native segwit not going to be supported in this PR?

Reply:
yes we are upgrading and defaulting to native for this release